frontends/ol-utilities/src/components/MITLogoLink.tsx

```diff
@@ -1,16 +1,17 @@
 import React from "react"
 
 const PUBLIC_URL = process.env.PUBLIC_URL || ""
-const MIT_LOGO_URL = `${PUBLIC_URL}/static/images/mit-logo-transparent.svg`
+const HOME_URL = `${PUBLIC_URL}/`
+const MIT_LOGO_URL = `${PUBLIC_URL}/static/images/mit-logo-transparent4.svg`
```

Comment:
Should the image include the text? In Figma it looks to be using text with font size 15px (we'd need another config in the theme typography for this).
https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=207-1834&m=dev

Reply:
Ah makes sense. Can probably ignore my other comment about positioning.